be/src/common/configbase.cpp

```diff
@@ -321,6 +321,7 @@ Status set_config(const std::string& field, const std::string& value) {
     UPDATE_FIELD(it->second, value, int32_t);
     UPDATE_FIELD(it->second, value, int64_t);
     UPDATE_FIELD(it->second, value, double);
+    UPDATE_FIELD(it->second, value, std::string);
```

Comment:
Maybe stringare not thread safe to change dynamically.

Reply:
have been solved